spec/controllers/state_file/questions/nj_dependents_health_insurance_controller_spec.rb

```diff
@@ -15,10 +13,10 @@
       let(:intake) { create :state_file_nj_intake, :df_data_two_deps }
       
       context "and did not have a health insurance requirement exception and all members had health insurance" do
-        it "does not show" do
+        it "shows" do
           allow_any_instance_of(StateFileNjIntake).to receive(:has_health_insurance_requirement_exception?).and_return(false)
           allow_any_instance_of(StateFileNjIntake).to receive(:eligibility_all_members_health_insurance_no?).and_return(false)
```

Comment:
[dust] - I think these two lines could optionally be removed now like they are for the no dependents scenario above, right?

```
allow_any_instance_of(StateFileNjIntake).to receive(:has_health_insurance_requirement_exception?).and_return(false)
          allow_any_instance_of(StateFileNjIntake).to receive(:eligibility_all_members_health_insurance_no?).and_return(false)
```